dev-cli

```diff
@@ -4,4 +4,10 @@ VERSION="local"
 LDFLAGS="-X 'github.com/mesg-foundation/core/version.Version=$VERSION'"
 
 go build -o ./bin/cli -ldflags="$LDFLAGS" interface/cli/main.go
-./bin/cli $@
+
+mesg_port=50052
+endpoint="localhost:$mesg_port"
+if [[ $DOCKER_HOST != "" ]]; then
+  endpoint="$(echo $DOCKER_HOST | awk '{while ( match($0,/([0-9]+\.){3}[0-9]+/) ) { print substr($0,RSTART,RLENGTH); $0=substr($0,RSTART+RLENGTH) } }'):$mesg_port"
```

Comment:
I feel this regex as super dirty..

Why not using `docker-machine ip default`?
source: https://docs.docker.com/machine/get-started/

Reply:
It just feel more natural to extract the value of `DOCKER_HOST` if we test for it but I agree using docker-machine is way simpler.
Fixed